### DISCUSSION TITLE
Favicon is not shown on Vercel

### DISCUSSION THREAD

**Discussion by Gumichocopengin8** [vercel/next.js#24886] [Help] [answered]
Hello, I uploaded Nextjs pages on Vercel, but favicon is not shown on Vercel (shown in localhost tho). 
Do you have any ideas to fix the issue? 
Here's the code and the web link
https://github.com/Gumichocopengin8/crate-trends/tree/develop/web 
https://crate-trends.vercel.app 

Thank you in advance

**Comment by timneutkens** [comment, marked as answer]
https://github.com/Gumichocopengin8/crate-trends/blob/develop/web/package.json#L7

It seems that you've set the root of the project to the `web` folder but the application is in `web/src/app`, potentially using a custom out directory? This causes `public` to be looked for as `web/public` instead of `web/src/app/public`, so the files are likely not used.

**Reply by Gumichocopengin8** [reply]
Thank you for answering. That makes sense!
Do you know why favicon was shown when I published it on Heroku? I believe that both of Vercel and Heroku use the same command to build.

**Reply by timneutkens** [reply]
You're using `next start` on Heroku, which you're passing the app path into.

**Reply by Gumichocopengin8** [reply]
I see. Thank you very much. That was very helpful.
